src/browser/utils.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+let inBrowser = false
+
+/**
+ * Set the value of if we are in the browser. Impacts {@link isInBrowser}.
+ */
+export function setInBrowser(value: boolean) {
+    inBrowser = value
+    return isInBrowser()
+}
+
+/**
+ * Return true if we are running in the browser, false otherwise.
+ */
+export function isInBrowser() {
```

Comment:
seems like this shoudl live in /shared/, not /browser/. Because non-browser code needs to call this.

Reply:
I was going through `vscode` source code and they structure their project like:

Example: https://github.com/microsoft/vscode/tree/main/src/vs/platform/request)
```
  {WHATEVER TOPIC}/
  - browser/
  - node/
  - common/
```
and this structure appears throughout the project in each topic folder.

---

So related to this comment, what do you think about creating a `src/common/` folder and then add in `browserUtils.ts`. So now we start using the `common` keyword for browser+node compatible code. And `shared` would instead be `utils` since "shared" may be confused with browser+node compatibility.

We'd document this new structure of course

Reply:
Sounds good to me!